tests/library/test_nutrition_order.py

```diff
@@ -0,0 +1,56 @@
+import json
+import os
+import unittest
+
+
+class TestNutritionKeyOrder(unittest.TestCase):
+
+    def test_nutrition_key_order(self):
+        correct_order = [
+            "type",
+            "calories",
+            "carbohydrateContent",
+            "cholesterolContent",
+            "fatContent",
+            "fiberContent",
+            "proteinContent",
+            "saturatedFatContent",
+            "servingSize",
+            "sodiumContent",
+            "sugarContent",
+            "transFatContent",
+            "unsaturatedFatContent",
```

Comment:
Generally most nutritional labels will present nutrients to match a standardised presentation order -- often `calories` first, followed by `fat`, then `sugars`.

When we re-ordered the recipe test JSON keys, we did that to assist the reviewability of the data, and I think that's what we should do here, too - so we needn't stick to alphabetical ordering.

I'd suggest:

```suggestion
            "servingSize",
            "calories",
            "fatContent",
            "saturatedFatContent",
            "unsaturatedFatContent",
            "transFatContent",
            "carbohydrateContent",
            "sugarContent",
            "proteinContent",
            "sodiumContent",
            "fiberContent",
            "cholesterolContent",
```

Energy, fat, carbohydrates, protein and salt are common requirements on nutritional labels; this includes those, with more detailed sub-elements adjacent to them where available, and a few other items included too.

Reply:
Hrm.  I mentioned salt, but `schema.org` only mentions `sodium` in the spec.  That's OK, but I'll try to review the data we have in this field soon to make sure it reports the expected measurement.